<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1344</width>
    <height>654</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chẩn đoán hình ảnh</string>
  </property>
  <widget class="QLabel" name="name">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>450</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="forward">
   <property name="geometry">
    <rect>
     <x>1056</x>
     <y>289</y>
     <width>61</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#forward{
background-color:transparent;
border-image:url(:/img/icons8-forward.png);
background:none;
border:none;
background-repeat:none;
}
#forward:pressed{
border-image:url(:/img/icons8-forward2.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="shortcut">
    <string>Right</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backward">
   <property name="geometry">
    <rect>
     <x>244</x>
     <y>291</y>
     <width>61</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#backward{
background-color:transparent;
border-image:url(:/img/icons8-back-50.png);
background:none;
border:none;
background-repeat:none;
}
#backward:pressed{
border-image:url(:/img/icons8-back.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="shortcut">
    <string>Left</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>10</y>
     <width>251</width>
     <height>631</height>
    </rect>
   </property>
   <property name="viewMode">
    <enum>QListView::IconMode</enum>
   </property>
  </widget>
  <widget class="QZoom" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>841</width>
     <height>631</height>
    </rect>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContents</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="first_img">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>610</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Ảnh gốc</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>490</y>
     <width>221</width>
     <height>113</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QSlider" name="contrast">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum">
         <number>-127</number>
        </property>
        <property name="maximum">
         <number>127</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::NoTicks</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Độ tương phản</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="brightness">
        <property name="minimum">
         <number>-255</number>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="sliderPosition">
         <number>0</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::NoTicks</enum>
        </property>
        <property name="tickInterval">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Độ sáng</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="5,5">
      <property name="spacing">
       <number>21</number>
      </property>
      <property name="bottomMargin">
       <number>19</number>
      </property>
      <item>
       <widget class="QLineEdit" name="ctr_num">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="brn_num">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>graphicsView</zorder>
  <zorder>name</zorder>
  <zorder>listWidget</zorder>
  <zorder>first_img</zorder>
  <zorder>forward</zorder>
  <zorder>backward</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QZoom</class>
   <extends>QGraphicsView</extends>
   <header>qzoom.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
